.github/workflows/rust.yml

```diff
@@ -121,6 +121,8 @@ jobs:
         env:
           CC: "sccache clang"
           CXX: "sccache clang++"
+      - name: fetch params
+        run: forest-cli fetch-params --keys
```

Comment:
Doesn't this happen automatically? Do you want it to be a separate step for easier debugging?

Reply:
Yeah it might be easier to observe warnings and errors if there are regressions.